pkg/asset/installconfig/kubevirt/validation.go

```diff
@@ -21,6 +22,25 @@ func Validate(ic *types.InstallConfig, client Client) error {
 	return allErrs.ToAggregate()
 }
 
+// ValidateForProvisioning is called by PlatformProvisionCheck
+func ValidateForProvisioning(client Client) error {
+	hcUnstructured, err := client.GetHyperConverged(context.Background(), "kubevirt-hyperconverged", "openshift-cnv")
+	if err != nil {
+		return fmt.Errorf("failed to get resource openshift-cnv/kubevirt-hyperconverged, with error: %v", err)
+	}
+
+	enabled, found, err := unstructured.NestedBool(hcUnstructured.Object, "spec", "featureGates", "HotplugVolumes")
+	if err != nil {
+		return err
```

Comment:
Maybe need to add here readable relevant description to the error (would be displayed to user)

Reply:
An error here is something not expected. It happens if JSON is malformed or some other internal error.

Reply:
done